Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -1235,12 +1319,87 @@ def fetch_incidents(client: Client, fetch_time: str, fetch_limit: str, last_run:
+    """
+    job_id = args.get('job_id')
+    interval_in_seconds = arg_to_number(args.get('interval_in_seconds'))
+    demisto.debug(f'in process_search_command_with_polling function, {job_id=}')
+
+    if not job_id:  # if this is the first time
+        process_name = args.get('process_name', '')
+        process_hash = args.get('process_hash', '')
+        event_id = args.get('event_id', '')
+        query = args.get('query', '')
+        start_time = str(args.get('start_time', '1 day'))
+        end_time = str(args.get('end_time', ''))
+        limit = arg_to_number(args.get('limit'))
+        start = arg_to_number(args.get('start'))
+
+        response = client.create_search_process_request(process_name=process_name, process_hash=process_hash,
+                                                        event_id=event_id, query=query, limit=limit,
+                                                        start_time=start_time, end_time=end_time, start=start)
+        demisto.debug(f'got {response=}')
+        return PollResult(partial_result=CommandResults(readable_output=f"job_id is {response.get('job_id')}."),
+                          response=None,
+                          continue_to_poll=True,
+                          args_for_next_run={"job_id": response['job_id']} | args
+                          )
+
+    # this is not the first time, there is a job_id
+    response = client.get_search_process_request(job_id=args['job_id'])
+    if response.get('contacted'):
+        status = 'Completed' if response.get('contacted') == response.get('completed') else 'In Progress'
+    else:
+        status = None
+    if status == 'In Progress':
+        message = CommandResults(
+            readable_output=f"Checking again in {interval_in_seconds} seconds...")
+        return PollResult(
+            partial_result=message,
+            response=None,
+            continue_to_poll=True,
+            args_for_next_run={"job_id": job_id,
+                               **args})
+
+    elif status == 'Completed':
+        output = {'status': status, 'job_id': job_id, 'results': response.get('results')}
+        title = "Completed Search Results:"
+        headers = ["process_hash", "process_name", "device_name", "device_timestamp", "process_pid", "process_username"]
+        human_readable = tableToMarkdown(name=title, t=output.get('results'), removeNull=True, headers=headers)
+        message = CommandResults(outputs_prefix='CarbonBlackEEDR.SearchProcess',
+                                            outputs=output,
+                                            outputs_key_field='job_id',
+                                            raw_response=response,
+                                            readable_output=human_readable)
+        return PollResult(
+            response=message,
+            continue_to_poll=False)
+        
+    else:
+        raise DemistoException('Failed to run process search')
```

Comment:
Same here

Reply:
There is no status in this case. we only check is completed == contacted